app/assets/javascripts/admin/addon/templates/users-list-show.hbs

```diff
@@ -325,13 +325,13 @@
             {{#if this.showSilenceReason}}
               <div
                 class="directory-table__cell silence_reason"
-                title={{user.silence_reason}}
+                title={{this.stripHtml user.silence_reason}}
```

Comment:
I don't think this `title` attribute is useful when it is just repeating the content show on screen. I believe we can just use `aria-labelledby="silence-reason-value"`?

Reply:
It is useful when the reason is truncated. Agree, that it is not perfect when the content is the same.

<img width="1103" alt="Screenshot 2025-03-05 at 10 14 19 am" src="https://github.com/user-attachments/assets/81b1f7dd-65b7-41de-ae55-60b373ce624b" />